pkgs/top-level/all-packages.nix

```diff
@@ -8040,21 +8040,28 @@ in
     else
       openjdk11.override { headless = true; };
 
+  /* boot jdk needed for 13 */
+  openjdk13bootjdk = callPackage ../development/compilers/openjdk/12.nix {
```

Comment:
I would rather use a naming convention that is consistent with the rest of the bootstrap JDKs. Also, we should probably use a headless build for the bootstrap, unless that doesn't work.

Reply:
I don't really like keeping OpenJDK 12 just for bootstrapping. AdoptOpenJDK provides OpenJDK 13 builds now, so I would rather we use those when possible to be consistent. The problem is that they don't provide i686 builds, but we should probably handle that using our own bootstrap tarball like we do for other OpenJDK versions. See #38097 for the last time this was done.

I'm willing to help with this, but I don't have the ability to upload bootstrap tarballs to `tarballs.nixos.org`.